### DISCUSSION TITLE
Manage issue labels

### DISCUSSION THREAD

**Discussion by kangz12345** [snu-quiqcl/qiwis#68] [Ideas]
Currently, there are 10 labels:
- app
- backend
- bug
- database
- documentation
- enhancement
- frame
- question
- refactor
- swift

However, I felt the labels are somehow not suitable.
For example, there is no `bus` label, and `backend` is not a part of `swift` (but of the toy model).

Therefore, I want to suggest a new set of labels:
- bug
- documentation
- enhancement
- **example**: About the example codes.
- **good first issue**: Good for newcomers.
- question
- refactor
- **repository**: About this git or GitHub repository.
- **style**: Code format or conventions.
- **test**: About testing.

Please leave comments if you have questions or opinions.

**Comment by kangz12345** [comment]
@BECATRUE @Ahrrri for notification

**Reply by BECATRUE** [reply]
Thanks for notification!

I have just a suggestion. How about adding `style`.
Like [this](https://doublesprogramming.tistory.com/256), I know that it is common to distinguish `documentation`, `refactor`, and `style`.

**Reply by kangz12345** [reply]
Very good idea! I added it to the suggested label list.

**Reply by BECATRUE** [reply]
Good! If there is no more suggestion, please update it.

**Reply by kangz12345** [reply]
I updated the labels and applied them to existing issues and PRs.
